torch/__init__.py

```diff
@@ -1457,17 +1472,25 @@ def _check_tensor_all(cond, message=None):  # noqa: F811
 
 # For Python Array API (https://data-apis.org/array-api/latest/API_specification/constants.html) and
 # NumPy consistency (https://numpy.org/devdocs/reference/constants.html)
-from math import e, nan , inf , pi
+from math import e, inf, nan, pi
 newaxis: None = None
 __all__.extend(['e', 'pi', 'nan', 'inf', 'newaxis'])
 
 ################################################################################
 # Define Storage and Tensor classes
 ################################################################################
 
-from ._tensor import Tensor
-from torch import storage as storage
-from .storage import _StorageBase, TypedStorage, _LegacyStorage, UntypedStorage, _warn_typed_storage_removal
+from torch._tensor import Tensor  # usort: skip
```

Comment:
Is there a lint for usort now?

Reply:
`usort` is part of `ufmt`. The `ufmt` command runs `usort` and `black`.